src/main/java/org/codehaus/plexus/util/xml/PrettyPrintXMLWriter.java

```diff
@@ -228,8 +228,10 @@ private static String escapeXml(String text) {
 
     private static final Pattern lowers = Pattern.compile("([\000-\037])");
 
+    private static final Pattern lowersText = Pattern.compile("([\000-\010\013-\014\016-\037])");
```

Comment:
Personally I find hexadecimal codes easier to follow than octal. I had to look these up. 

Also "lowersText" isn't obvious to me. Perhaps "illegalC0Characters"